source/plugin/event/index.rst

```diff
@@ -0,0 +1,39 @@
+===================
+Working with Events
+===================
+
+Sponge provides a system to:
+
+- Listen for events
+- Fire events
+
+Overview
```

Comment:
I'd say we should get rid of the ``Overview`` heading and the list above. Just some introductory sentences should suffice.